src/orchestrator-process/src/lib.rs

```diff
@@ -426,6 +436,17 @@ async fn supervise(
         match cmd.spawn() {
             Ok(process) => {
                 let status = KillOnDropChild(process).0.wait().await;
+                if panic_on_child_crash {
+                    if let Ok(status) = status {
+                        match status.signal() {
+                            Some(6) | Some(7) | Some(11) => {
```

Comment:
@benesch . Do you want the signals to be named constants here? I need to pull in yet another crate in order to get named constants though.

Reply:
Yes, pulling in [`nix::Signal`](https://docs.rs/nix/latest/nix/sys/signal/enum.Signal.html) would be great.

Reply:
I had to use i32 constants from the libc crate  because [std](https://doc.rust-lang.org/stable/std/index.html)::[os](https://doc.rust-lang.org/stable/std/os/index.html)::[unix](https://doc.rust-lang.org/stable/std/os/unix/index.html)::[process](https://doc.rust-lang.org/stable/std/os/unix/process/index.html)::[ExitStatusExt](https://doc.rust-lang.org/stable/std/os/unix/process/trait.ExitStatusExt.html#) returns a i32 . The nix::Signal enum has lost its `from_c_int` method and therefore there is no way to compare nix::Signal to i32  without `unsafe`. I hope this is acceptable.